### PR TITLE
Optimised images v1

### PR DESCRIPTION
**Pull request by rishikeshDpp**

### CODE REVIEW

File: src/components/Chat/constants.ts [hunk 1]
```diff
@@ -1,24 +1,7 @@
-import dfinityimg from "images/avatar/01.png";
-import ezraimg from "images/avatar/02.png";
-import webthreeimg from "images/avatar/03.png";
-import boxydudeimg from "images/avatar/04.png";
-import quantumcomputingimg from "images/avatar/05.png";
-import sathoshiimg from "images/avatar/06.png";
-import sonicimg from "images/avatar/07.png";
 import { X_Handle } from "src/constants";
 
 export const DEFAULT_AGENT_ID = "2";
```
Comment (mohitrajane) [inline src/components/Chat/constants.ts:3]: can we delete this too, don't think we use it anywhere

File: src/components/CreateAgent/TemplateSore.tsx
Comment (mohitrajane) [inline src/components/CreateAgent/TemplateSore.tsx:1]: is this a duplicate file? is that why we are deleting it?

File: src/components/CreateAgent/View.tsx
Comment (mohitrajane) [inline src/components/CreateAgent/View.tsx:1]: is this a duplicate file? is that why we are deleting it?

**Review by mohitrajane — CHANGES_REQUESTED** [review]

File: src/components/Chat/constants.ts (hunk 1 above)
Comment (rishikeshDpp) [reply src/components/Chat/constants.ts:3]: Which line?

File: src/components/CreateAgent/constants.ts [hunk 2]
```diff
@@ -1,49 +1,8 @@
...
-    id: uuidv4(),
-    name: "Morgan Freeman",
-    description:
-      "Create a professional AI agent in minutes. Simply upload related documents or web links, and it will become an expert in your field.",
-    imageUrl: morgan,
-    // tag: "Knowledge-base Genius",
-  },
-  {
-    id: uuidv4(),
-    name: "Albert Einstein",
```
[the remark this answers is not in the document]
Comment (rishikeshDpp) [reply src/components/CreateAgent/constants.ts:23]: Yep

File: src/components/Chat/constants.ts (hunk 1 above)
Comment (rishikeshDpp) [reply src/components/Chat/constants.ts]: Removed DEFAULT_AGENT_ID

**Review by mohitrajane — APPROVED** [review]

**Pull request merged**